### PR TITLE
Create the guild model

### PR DESCRIPTION
**Pull request by guillermosanta**

### CODE REVIEW

File: api/migrations/20210812174148-create-guild.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("guilds", {
+      id: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      prefix: {
+        defaultValue: ".",
```
Comment (chungthuang) [inline api/migrations/20210812174148-create-guild.js:10]: What does prefix mean and why do we pick this as the default.

File: api/migrations/20210812174148-create-guild.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      threadInactivityTime: {
+        defaultValue: 1000 * 60 * 60 * 24 * 7,
+        type: Sequelize.INTEGER,
+      },
+      moderationServer: {
+        type: Sequelize.STRING,
+      },
+      createdAt: {
+        allowNull: false,
```
Comment (chungthuang) [inline api/migrations/20210812174148-create-guild.js:33]: Should this default to now, the current timestamp?

File: api/migrations/20210812174148-create-guild.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("guilds", {
+      id: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
```
Comment (chungthuang) [inline api/migrations/20210812174148-create-guild.js:8]: Should this be an UUID?

File: api/migrations/20210812174148-create-guild.js (hunk 1 above)
Comment (guillermosanta) [reply api/migrations/20210812174148-create-guild.js:10]: The prefix is what users need to add in front of the message to indicate that it is a command. For example `.help`, `.balance` etc.
We want the prefix to be customizable, so there will be a command that will allow mods to change the prefix in their correspondent guild.

File: api/migrations/20210812174148-create-guild.js (hunk 2 above)
Comment (guillermosanta) [reply api/migrations/20210812174148-create-guild.js:33]: Sequelize automatically manages and adds this fields.  
```By default, Sequelize automatically adds the fields createdAt and updatedAt to every model, using the data type DataTypes.DATE. Those fields are automatically managed as well - whenever you use Sequelize to create or update something, those fields will be set correctly. The createdAt field will contain the timestamp representing the moment of creation, and the updatedAt will contain the timestamp of the latest update.```

File: api/migrations/20210812174148-create-guild.js (hunk 3 above)
Comment (guillermosanta) [reply api/migrations/20210812174148-create-guild.js:8]: The id of a guild is a Snowflake which is just a long string. It is not an id I generate myself.
https://discord.js.org/#/docs/main/stable/typedef/Snowflake

File: api/migrations/20210812174148-create-guild.js (hunk 2 above)
Comment (chungthuang) [reply api/migrations/20210812174148-create-guild.js:33]: Oh I see. It's been a while since I looked at sequailize.

**Review by chungthuang — APPROVED** [review]

**Pull request merged**